build.xml

```diff
@@ -3,7 +3,7 @@
 <project name="tideman" default="build">
 
     <target name="build" depends="lint,phpstan,phpcs,phpunit" />
-    <target name="build-ci" depends="lint,phpstan,phpcs,phpunit-ci" />
```

Comment:
I thought it would be good to print logs by default on CI, and to have them be opt-in for devs because the test result summary and the code coverage summary are super convenient to have at the bottom. Ideally I would want output to be ordered as follows, but I don't know how to print the test result summary separately from the actual running of the tests.

Ideal order:
...some stuff...
Contents of test log file
Test runner output (the dots and E's and W's)
Test coverage summary stats.
Build status